labs/b4.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Beginner Lab 4 - Debian, packages, compiling software
+layout: lab
+---
+
+## What is a distribution?
+
+A Linux distribution is a combination of the Linux kernel, package management system, drivers, scheduler,
+ and other software. They are the operating systems that use Linux as a kernel. 
+
+## What should I use then?
+
+Currently, there are around 300 actively developed distributions listed on DistroWatch. It might be a bit overwhelming with so many choices. Popular distributions include Debian, Ubuntu, Arch Linux, Fedora, and Red Hat Linux, but in this course, we’ll only be using Debian.
+
+### What is Debian?
+
+Debian is a distribution developed by the Debian Project beginning in 1993. This distribution strives to provide a completely free (libre) software experience. Many other distributions are based on Debian as Debian releases are usually very stable. One such distribution is Ubuntu, which is very popular and has spawned its many of its own derivatives. What unifies all of Debian-based distributions is the usage of the DEB package format and the dpkg package manager.
```

Comment:
I'd link to debian.org and add a comma before beginning, as well as define "libre" and "stable." Also, "its many of its own"?